ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -287,97 +120,104 @@ export class ImportedDocuments extends React.Component {
     );
   };
 
-  filteredRecords = () => {
-    const { statistics, data, selectedResult } = this.state;
-    const selectedFilterFunc = statistics[selectedResult].filterFunction;
+  filteredRecords = data => {
+    const { selectedResult } = this.state;
+    const selectedFilterFunc = this.statistics[selectedResult].filterFunction;
+
+    if (!data) return null;
```

Comment:
It should return empty list?
```suggestion
    if (!data) return [];
```
Can we check this before calling it and remove the `if` from here? It looks weird to call this func if data is empty...

Reply:
That is not really possible because that if would block the entire render method. Now it will just show a loading screen for a quick moment. I returned the null because we have a check for that, even returning an empty array would give errors anyway.